config/dbConnect.js

```diff
@@ -0,0 +1,18 @@
+const { Sequelize } = require ("sequelize");
+const sequelize = new Sequelize(process.env.DATABASE, process.env.USERNAME, process.env.PASSWORD, {
+    host: 'localhost',
```

Comment:
Move localhost to an environmental variable and then update the host to reflect that.